h/views/admin_groups.py

```diff
@@ -169,6 +194,10 @@ def _update_appstruct(self):
             'group_type': group.type,
             'name': group.name,
             'origins': [s.origin for s in group.scopes],
+
+            # Once `group.organization` becomes non-nullable we can simplify
```

Comment:
Do we have a standard keywording (e.g. `TODO` or `FIXME`) for this? It might be helpful to drop that into the comment to catch the eye later.